notebooks/README.md

```diff
@@ -6,9 +6,18 @@ The tutorial consists of the following steps:
 - Download the necessary models
 - Convert the models to OpenVINO IR.
 - Load models to OpenVINO engine 
+- Perform object detection using yolov8, text detection and recognition. 
 - Live demo
 
-![automated checkout](output.jpg)
+![automated checkout](automated.gif)
```

Comment:
did you change the earlier image ?
Am not seeing the objects & text detected a in earlier commit. This was good. 
https://github.com/intel-retail/automated-self-checkout/blob/7892ba887a6b42e8842e3ec7c8a98cfd38556505/notebooks/automated-gif.gif

Reply:
good catch!, this new gif was not generated correctly. I will modify it